packages/ndla-notion/src/LicenseByline/LicenseDescriptionList.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from "@emotion/styled";
+import { spacing } from "@ndla/core";
+import { getLicenseRightByAbbreviation } from "@ndla/licenses";
+import StyledList from "./LicenseList";
```

Comment:
Eksportert verdi heiter vel LicenseList.
```suggestion
import LicenseList from "./LicenseList";
```